sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/HandleItem.java

```diff
@@ -279,7 +279,7 @@ public List<AccessRight> getAccessRightList() {
      * @param accessRightList the accessRightList value to set.
      * @return the HandleItem object itself.
      */
-    public HandleItem setAccessRightList(List<AccessRight> accessRightList) {
+    public HandleItem setAccessRightList(List<ShareFileHandleAccessRights> accessRightList) {
```

Comment:
Method name should be updated to `setAccessRights` and param name should be `accessRights`. Also, update the javadocs.